docs/standard/serialization/system-text-json/migrate-from-newtonsoft.md

```diff
@@ -68,15 +68,16 @@ The following table lists `Newtonsoft.Json` features and `System.Text.Json` equi
 | `DefaultContractResolver` to ignore properties        | ✔️ [DefaultJsonTypeInfoResolver class](#conditionally-ignore-a-property) |
 | Polymorphic serialization                             | ✔️ [[JsonDerivedType] attribute](#polymorphic-serialization) |
 | Polymorphic deserialization                           | ✔️ [Type discriminator on [JsonDerivedType] attribute](#polymorphic-deserialization) |
+| Deserialize string enum value                         | ✔️ [Deserialize string enum values](#deserialize-string-enum-values) |
+| `MissingMemberHandling` global setting                | ✔️ [Handle missing members](#handle-missing-members) |
+| Add to collections without setters                    | ✔️ [](#add-to-collections-without-setters) |
```

Comment:
We should probably generalize this beyond just collections
```suggestion
| Populate properties without setters                    | ✔️ [](#add-to-properties-without-setters) |
```

Reply:
Will address in separate PR.